pkg/k8s/version/version.go

```diff
@@ -45,6 +45,9 @@ type ServerCapabilities struct {
 	// EndpointSlice is the ability of k8s server to support endpoint slices
 	EndpointSlice bool
 
+	// EndpointSliceV1 is the ability of k8s server to support endpoint slices v1
```

Comment:
Can u briefly add a bit more description like what version of k8s it was introduced, like below `LeasesResourceLock` & `APIExtensionsV1CRD`? It makes it much easier to find useful info about thecapabilities